services/web/server/src/simcore_service_webserver/socketio/messages.py

```diff
@@ -62,12 +62,12 @@ async def send_message_to_user(
     user_id: UserID,
     message: SocketMessageDict,
     *,
-    ignore_queue: bool,
+    ignore_queue: bool = False,
```

Comment:
if `ignore_queue=True` is nto used anywhere after this PR ... we could perhaps remove it?

![image](https://github.com/user-attachments/assets/eb0e0380-8980-4d2f-84ed-d7d4c3c51c0a)